jobs/dd-agent/templates/bin/forwarder_ctl

```diff
@@ -18,11 +18,9 @@ case ${1:-help} in
   start)
     echon_log "Starting Datadog Agent $COMPONENT: "
     pid_guard $PIDFILE "dd-agent-$COMPONENT"
-    # Load python library path for dynamic libs
-    ldconf "$LD_LIBRARY_PATH"
     (
         {
-            exec chpst -v -u vcap:vcap python "$DD_FORWARDER" --use_simple_http_client=1
+            exec chpst -v -u vcap:vcap $DD_AGENT_PYTHON "$DD_FORWARDER" --use_simple_http_client=1
```

Comment:
since we're using a packaged agent now, with all the deps included (including the ones compiled at build time like `tornado`), we shouldn't force `--use_simple_http_client=1` here